data/mods/gen3colosseum/scripts.ts

```diff
@@ -0,0 +1,503 @@
+export const Scripts: ModdedBattleScriptsData = {
+	inherit: 'gen4',
```

Comment:
```suggestion
	inherit: 'gen4',
```

If you inherit `gen3` you'll be able to layer your changes on top, instead of needing to duplicate everything `gen3` does.

Reply:
Oh, I will fix everything to switch to that. That is way easier

Reply:
It applies to everything, not just Scripts.

Reply:
(For anyone else reading and wondering, I had to delete my dist to make this work)